pyFAST/fastfarm/examples/Ex3_FFarmCompleteSetup.py

```diff
@@ -20,100 +20,112 @@ def main():
     # -----------------------------------------------------------------------------
 
     # ----------- Case absolute path
-    path = '/complete/path/of/your/case'
+    path = 'path to case'

```

Comment:
Unfortunately, Examples 2,3 and 4 are not "standalone" at the moment and cannot be run. 

Still, it might be best (for a git history perspective), if you only include in this example the changes relevant to your commit (do not change hub height, frequency, number of turbines, paths, tmax, etc). 

To separate the batch and slurm code, you can add a flag at the top of the script, for instance:
```python
 scheduler='batch      # Which jobs scheduler to use ('batch' or 'slurm')
 #scheduler='slurm'  
```
and then for the places where your changes are relevant to batch:
```python
if scheduler=='batch':
    case.copyTurbineFilesForEachCase()
    case.FF_bat_file()
elif scheduler=='slurm':
    # etc
```

Reply:
Done